src/Frontend/Core/Engine/Breadcrumb.php

```diff
@@ -59,7 +59,7 @@ private function addBreadcrumbsForPages(array $pages): void
 
         array_map(
             function (array $breadcrumb) {
-                $this->addElement($breadcrumb['title'], $breadcrumb['url']);
+                $this->addElement($breadcrumb['title']);
             },
```

Comment:
won't this remove the url for all pages if they are nested?

Reply:
Maybe, need to check...

Reply:
You are right. This hack removes in sub-pages to :(
Pls remove this PR.